lib/Localization/LocalizationFormat.cpp

```diff
@@ -98,7 +98,8 @@ LocalizationProducer::getMessageOr(swift::DiagID id,
   if (localizedMessage.empty())
     return defaultMessage;
   llvm::StringRef diagnosticName(diagnosticNameStrings[(unsigned)id]);
-  if (localizationSaver && printDiagnosticName) {
+  if (printDiagnosticName) {
+    assert(localizationSaver);
```

Comment:
@owenv I'm checking on the value of `printDiagnosticName` and asserting on `localizationSaver` to check if both are present. That what you meant, right?

Reply:
yeah, exactly!